zp-relayer/validateTx.ts

```diff
@@ -297,3 +303,8 @@ export async function validateTx(
     await checkAssertion(() => checkScreener(userAddress, traceId))
   }
 }
+
+export async function validateDirectDeposit(dd: DirectDeposit) {
```

Comment:
Should we double check the direct deposit status over here, by fetching it from the contract?

Reply:
If we assume a reliable watcher, then worker would process only uncompleted deposits. But, probably, we could add status validation as a sanity check.